packages/stryker-mocha-runner/README.md

```diff
@@ -37,15 +37,53 @@ module.exports = function (config) {
         // ...
         mochaOptions: {
             // Optional mocha options
+            files: [ 'test/*.js' ]
             opts: 'path/to/mocha.opts',
             ui: 'bdd',
+            timeout: 3000,
             require: [ /*'babel-register' */],
             asyncOnly: false
         }
     });
 }
 ```
 
-You can pass mocha options in 2 ways: either specify a mocha.opts file or configure `mochaOptions` in stryker.conf.js. When both are specified, the values in stryker.conf.js will overrule the values in your mocha.opts file.
+### `mochaOptions.files` [`string` or `string[]`]
+
+Default: `'test/*.js'`
```

Comment:
Wait, shouldn't this be `test/**/*.js`? Or does the globbing implementation work differently?

Reply:
It is the default in mocha configuration. That's why I've chosen it as a default. At the same time, I think it is a silly default, so let's make it `test/**/*.js` 👍